GVRf/Framework/jni/objects/components/transform.cpp

```diff
@@ -43,6 +42,16 @@ void Transform::invalidate() {
             (*it)->transform()->invalidate();
         }
     }
+    // scale rotation_ if needed to avoid overflow
```

Comment:
I am not quite sure but my concern is that if it is possible that when we call getModelMatrix(), actually the over/under flow has already happened by some calls like setRotation or even setModelMatrix and we don't have a way to know at getModelMatrix().

However I think to do the check on each invalidate is a little bit waste of performance. If it is possible to find a way that only when we change the rotation_ then we do such check.